sirepo/sim_data/__init__.py

```diff
@@ -655,8 +655,13 @@ def _organize_example(cls, data):
             dm.simulation.folder = '/Examples'
 
 
-def uid_from_jid(jid):
-    return jid.split(_JOB_ID_SEP)[0]
+def split_jid(jid):
+    """Reverse of parse_jid. Take a jid and split it into its parts"""
+    s = jid.split(_JOB_ID_SEP)
```

Comment:
you want to use `zip`:

```py
return PKDict(zip(
    ('uid', 'simulation_id', 'compute_model'), 
    jid.split(_JOB_ID_SEP),
))
```

Reply:
https://github.com/radiasoft/sirepo/pull/2534/commits/c462bafae19f5d6ad2d76998d1460fbfccf747b5